### DISCUSSION TITLE
Input error when excecute ww3_strt from bash script

### DISCUSSION THREAD

**Discussion by DiegoGutierrez2210** [NOAA-EMC/WW3#873] [Q&A]
Hi guys

I'm running simulations everyday for a forecast tool with WW3, but recently I have this error:
**cat ww3_strt.out
 
                  *** WAVEWATCH III  Initial conditions ***   
               ===============================================

 *** WAVEWATCH III ERROR IN W3STRT : 
     ERROR IN OPENING INPUT FILE
     IOSTAT =   29

 w3servmd MPI_ABORT, IEXIT=          10
 w3servmd UNIT missing
 w3servmd MSG missing
 w3servmd FILE missing
 w3servmd LINE missing
 w3servmd COMM missing**

This is using CALM condition, and it is strange error because when I excecute it manually the error disappear. And, for operational reasons, WW3 simulation have to be in bash to runs every day at 6am. Can someone help me please? 

Regards

**Comment by ukmo-ccbunney** [comment]
The error suggests that the ww3_strt program cannot find the `ww3_strt.inp` or `ww3_strt.nml` input file.
Are you sure the script is running where you think it is? Are you submitting it via a scheduler such as slurm or PBS (I notice that it is an MPI job)? Often this will not automatically be run in the directory you submit from...

Have you tried running with 
`set -x`
to get extra Bash debugging info?
